platform/qt/src/qmapboxgl.cpp

```diff
@@ -1005,23 +1225,55 @@ void QMapboxGL::addLayer(const QVariantMap &params)
     d_ptr->mapObj->addLayer(std::move(*layer));
 }
 
+/*!
+    Removes the layer \a id.
+*/
 void QMapboxGL::removeLayer(const QString& id)
 {
     d_ptr->mapObj->removeLayer(id.toStdString());
 }
 
-void QMapboxGL::addImage(const QString &name, const QImage &sprite)
+/*!
+    Adds the \a image with the identifier \a id that can be used
```

Comment:
If the name is the same, the image get replaced only if the dimensions are the same (I just looked it up). I'm going to document this.